specification/consumption/resource-manager/Microsoft.Consumption/preview/2019-05-01-preview/consumption.json

```diff
@@ -0,0 +1,3186 @@
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "$skiptoken",
+            "description": "Skiptoken is only used if a previous operation returned a partial result. If a previous response contains a nextLink element, the value of the nextLink element will include a skiptoken parameter that specifies a starting point to use for subsequent calls.",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "$top",
+            "description": "May be used to limit the number of results to the most recent N usageDetails.",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 1000
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/metricParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/UsageDetailsListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.Consumption/usageDetails/download": {
+      "post": {
+        "tags": [
+          "UsageDetails"
+        ],
+        "x-ms-examples": {
+          "UsageDetailsDownload": {
+            "$ref": "./examples/UsageDetailsDownload.json"
+          },
+          "UsageDetailsDownloadByMetricUsage": {
+            "$ref": "./examples/UsageDetailsDownloadByMetricUsage.json"
+          }
+        },
+        "operationId": "UsageDetails_Download",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "description": "Download usage details data.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/scopeUsageDetailsParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/metricParameter"
+          },
+		  {
+            "$ref": "#/parameters/languageParameter"
```

Comment:
Why are you introducing an explicit language parameter to this API? All ARM requests are supposed to respect the Accept-Language header as the requested language from the user.

https://github.com/Azure/azure-resource-manager-rpc/blob/70d10c715342149a3be94ebdc75ebde5ea0c9354/v1.0/common-api-details.md#client-request-headers

Reply:
In usageDetailsDownload the API caller needs to pass" ln=en" for instance as a url parameter. Then it is required. We are giving the URL to the CSV file we need to prepare the file based on the caller language specific. In previous PR it was not there, then I added it.

Reply:
I understand this is how your API was documented. To be compliant with ARM RPC, you should just use the Accept-Language header from the caller. Having an explicit 'ln' query parameter isn't needed. If you didn't make this change, whats the expected behavior when these 2 languages aren't the same?

Reply:
@ryansbenson: Thank you for the review and I removed language parameter from usage detail download and related examples based on your suggestion to be compatible with ARM.

Reply:
@ryansbenson could you please review this again, I addressed your comments.